circle.yml

```diff
@@ -17,7 +17,7 @@ deployment:
 
 test:
   override:
-    - bundle exec rake test TESTOPTS="--ci-dir=$CIRCLE_TEST_REPORTS":
+    - bundle exec rails test TESTOPTS="--ci-dir=$CIRCLE_TEST_REPORTS":
```

Comment:
swap this back to rake, create a todo as to why `rails test` is eating mud in CI